site.srht.shrimple.Ouch.json

```diff
@@ -0,0 +1,57 @@
+{
+    "id": "site.srht.shrimple.Ouch",
```

Comment:
https://shrimple.srht.site/ says 404 not found, how do we verify that you own the domain?

Reply:
Ah nvm, it is code hosting.